infra/resources/prod/locals.tf

```diff
@@ -73,6 +73,22 @@ locals {
       {
         name                  = "AppInsightsConnectionString"
         key_vault_secret_name = "AppInsightsConnectionString"
+      },
+      {
+        name                  = "AuthProfileApiKey"
+        key_vault_secret_name = "AuthProfileApiKey"
+      },
+      {
+        name                  = "MailupUsername"
+        key_vault_secret_name = "MailupUsername"
+      },
+      {
+        name                  = "MailupSecret"
+        key_vault_secret_name = "MailupSecret"
+      },
+      {
+        name                  = "MailSender"
+        key_vault_secret_name = "MailSender"
       }
```

Comment:
is `MailSender` a secret value? @grausof

Reply:
nope

Reply:
then you should remove it from here and set as plain value @christianatzeni

Reply:
Okay! Do we already have a value for this variable?

Reply:
`IO - l'app dei servizi pubblici <no-reply@io.italia.it>`